### ISSUE TITLE
Is there any way to know last execution result of a task?

### ISSUE THREAD

**Issue by cbelizone2y** [kagkarlsson/db-scheduler#147]
Hi,

We are using the library and we would like to know if there is any way to retrieve the status of the last execution of a specific task. Checking the api and documentation we see nothing.

**Comment by kagkarlsson** [comment]
I think goes into the pool of improvements that could be made to the `getSchedulesExecutions` api on `SchedulerClient`.

Currently, there is this method on `Scheduler`
```java
    public List<Execution> getFailingExecutions(Duration failingAtLeastFor) {
        return taskRepository.getExecutionsFailingLongerThan(failingAtLeastFor);
    }
```

It will return executions that are in a "failing" state, i.e. last run was a failure, that that was at least `failingAtLeastFor` time ago. Intended to be used for alerting

**Comment by kagkarlsson** [comment]
This is being addressed in #155 I think. Please have a look to see if that satisfies your requirement.

Specifically adding methods to `ScheduledExecution`:

```
    public Instant getLastSuccess()
    public Instant getLastFailure()
    public int getConsecutiveFailures()
    public boolean isPicked()
    public String getPickedBy()
```

**Comment by kagkarlsson** [comment]
Fixed in `v9.2`
